### ISSUE TITLE
[R] Increase performance with caching

### ISSUE THREAD

**Issue by stefanreifenberg** [bigomics/omicsplayground#87]
- [ ] cache tables
- [ ] cache plots

Further reading:
https://shiny.rstudio.com/articles/caching.html

TLDR:
What is caching?

In Shiny 1.6.0, reactives and render functions can cache their values, using the [bindCache()](https://shiny.rstudio.com/reference/shiny/latest/bindCache.html) function. In other words, the application will do the computation with a given set of inputs once, and then after that, if those inputs are seen again, it can simply retrieve the value saved in the cache, instead of having to compute it again.

When does caching help?

Because caching saves computed values for use later, it is most effective when the same computation is done multiple times. If, for example, you have a data dashboard where many users see the same data and the same plots of the data, then with caching, the data processing and plotting can happen once, and after that, the results simply have to be retrieved from the cache.

This document covers the following:

[Using bindCache()](https://shiny.rstudio.com/articles/caching.html#using-bindcache): [bindCache()](https://shiny.rstudio.com/reference/shiny/latest/bindCache.html) can be used with reactive() and render functions, like renderPlot() and renderText().
[Using memoise()](https://shiny.rstudio.com/articles/caching.html#using-memoise): memoise() can be used with “normal” R functions.
[Cache scoping](https://shiny.rstudio.com/articles/caching.html#cache-scoping)
[Best practices for deployed applications](https://shiny.rstudio.com/articles/caching.html#best-practices-for-deployed-applications)
[Frequently Asked Questions](https://shiny.rstudio.com/articles/caching.html#faq)

**Comment by ivokwee** [comment]
The problem is that each user has his/her mostly her own data set. So caching across users would probably not work in the general case.
